yarn.lock

```diff
@@ -16180,16 +16180,10 @@ oncokb-styles@~0.1.2:
   resolved "https://registry.yarnpkg.com/oncokb-styles/-/oncokb-styles-0.1.2.tgz#8b26c0a0829787cdc1b595d3a021b3266607102b"
   integrity sha512-tuy5s3qFxgf1ogMATQSRPNgLlAMrvOOTCAN1dm/wJ+VZoStbJ7g36/qHwc99UPfh3vrB05broLodF+k58p5tUw==
 
-oncoprintjs@4.2.3:
```

Comment:
Not sure why this is deleted but the oncoprintjs is still in the yarn.lock

Reply:
probably this version not used anymore?

Reply:
Yeah I suppose so.